AUTHORS

```diff
@@ -26,8 +27,10 @@ Aditya Jain <adityajain.783@gmail.com>
 Ajo John <ajojohn555@gmail.com>
 Akshay Anand <akshayanand681@gmail.com>
 Alex Gower <alexanderpgower@gmail.com>
+Alex Lee <alexanderwlee@gmail.com>
```

Comment:
I think we reverted Alex's only PR. Is this correct (could you verify)? If so, he should not be in this list (or others).

Reply:
Yes, you are right. I'll remove him for now. (Thanks for catching this.)

Reply:
Done!